lib/nexpose/scan_template.rb

```diff
@@ -583,5 +583,20 @@ def enable_enhanced_logging=(enable)
       self.enable_debug_logging = enable
       self.aces_level = (enable ? 'full' : 'none')
     end
+
+    # Enable or disable windows service editor
+    # @param [Boolean] enable Enable or disable windows service editor
+    def windows_service_editor=(enable)
+      cifs_scanner = REXML::XPath.first(@xml, 'ScanTemplate/Plugins/Plugin[@name="java/CifsScanner"]')
+      param = REXML::XPath.first(cifs_scanner, './param[@name="windowsServiceEditor"]')
+      if param
+        param.text = (enable ? 1 : 0)
+      else
+        param = REXML::Element.new('param')
+        param.add_attribute('name', 'windowsServiceEditor')
+        param.text = (enable ? 1 : 0)
+        cifs_scanner.add_element(param)
+      end
```

Comment:
The return value is not consistent here (in the case of the parameter existing it is 1/0 but when it does not exist the added element is the return value). I'd expect the new value to be returned when the method exists.